Readme.md

```diff
@@ -31,10 +31,14 @@ Here's an example of what an app shows up as on the Plex web interface
 
 The rows in that example from top to bottom are `deviceName`, `version`, `product`, and `device`.
 
-### .query(uri)
+### .query(parameters)
```

Comment:
Should we call it options rather than parameters? Mainly of two reasons: (1) "options" are the defacto name for such objects in the JavaScript world, and (2) parameters in the context of plex-api sounds like URL parameters, at least that was what I thought at first, which is kind of misleading.